roles/ceph-common/handlers/main.yml

```diff
@@ -46,81 +20,23 @@
     state: restarted
   when:
     - socket.rc == 0
-    - use_systemd
     - osd_group_name in group_names
-    - ceph_release_num.{{ ceph_release }} > ceph_release_num.hammer
-
-- name: restart ceph osds on ubuntu
-  shell: |
-    for id in $(ls /var/lib/ceph/osd/ |grep -oh '[0-9]*'); do
-      initctl restart ceph-osd cluster={{ cluster }} id=$id
-    done
-  when:
-    - socket.rc == 0
-    - ansible_distribution == 'Ubuntu'
-    - not use_systemd
-    - osd_group_name in group_names
-
-- name: restart ceph mdss on ubuntu
-  command: initctl restart ceph-mds cluster={{ cluster }} id={{ ansible_hostname }}
-  when:
-    - socket.rc == 0
-    - ansible_distribution == 'Ubuntu'
-    - not use_systemd
-    - mds_group_name in group_names
-
-- name: restart ceph mdss
-  command: service ceph restart mds
-  when:
-    - socket.rc == 0
-    - ansible_distribution != 'Ubuntu'
-    - use_systemd
-    - mds_group_name in group_names
-    - ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis
 
 - name: restart ceph mdss with systemd
   service:
       name: ceph-mds@{{ mds_name }}
       state: restarted
   when:
     - socket.rc == 0
-    - use_systemd
     - mds_group_name in group_names
-    - ceph_release_num.{{ ceph_release }} > ceph_release_num.hammer
-
-- name: restart ceph rgws on ubuntu
-  command: initctl restart radosgw cluster={{ cluster }} id=rgw.{{ ansible_hostname }}
-  when:
-    - socketrgw.rc == 0
-    - ansible_distribution == 'Ubuntu'
-    - not use_systemd
-    - rgw_group_name in group_names
-
-- name: restart ceph rgws
-  command: /etc/init.d/radosgw restart
-  when:
-    - socketrgw.rc == 0
-    - ansible_distribution != 'Ubuntu'
-    - rgw_group_name in group_names
-    - ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis
-
-- name: restart ceph rgws on red hat
-  command: /etc/init.d/ceph-radosgw restart
-  when:
-    - socketrgw.rc == 0
-    - ansible_os_family == 'RedHat'
-    - rgw_group_name in group_names
-    - ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis
 
 - name: restart ceph rgws with systemd
```

Comment:
I think we can remove the "with systemd". Don't forget to change `generate_ceph_conf.yml` with the new name as well.